deepchem/feat/molecule_featurizers/mat_featurizer.py

```diff
@@ -0,0 +1,96 @@
+from deepchem.feat.base_classes import MolecularFeaturizer
+try:
+  import logging
+  import numpy as np
+  import pandas as pd
+  import torch
+  from rdkit import Chem
+  from rdkit.Chem import AllChem
+  from rdkit.Chem import MolFromSmiles
+  from sklearn.metrics import pairwise_distances
+  from torch.utils import datasets
+except:
+  raise ImportError("Required Modules not found.")
+
+use_cuda = torch.cuda.is_available()
+
+if use_cuda:
+  FloatTensor = torch.cuda.FloatTensor
+  LongTensor = torch.cuda.LongTensor
+  IntTensor = torch.cuda.IntTensor
+  DoubleTensor = torch.cuda.DoubleTensor
+else:
+  FloatTensor = torch.FloatTensor
+  LongTensor = torch.LongTensor
+  IntTensor = torch.cuda.IntTensor
+  DoubleTensor = torch.cuda.DoubleTensor
+
+
+class MATFeaturizer(MolecularFeaturizer):
+
+  def __init__(
+      self,
+      add_dummy_node=True,
+      one_hot_formal_charge=True,
+  ):
+    self.mol = mol
+    self.add_dummy_node = add_dummy_node
+    self.one_hot_formal_charge = one_hot_formal_charge
+
+  def OHVector(value, array):
```

Comment:
There's already a `one_hot_encode` function in `deepchem.utils.molecule_feature_utils`, use that one.

Reply:
Done! I've replaced all OHVector calls with the Deepchem one_hot_encode function and deleted the OHVector function.